internal/event/context_unexported_test.go

```diff
@@ -44,6 +45,28 @@ func Test_newSendCtx(t *testing.T) {
 			}(),
 			wantCancel: true,
 		},
+		{
```

Comment:
should we add an assertion after line #90 of:
`assert.True(ctx.Err() != context.DeadlineExceeded)`

Reply:
oooh, good catch! 👍

fixed!